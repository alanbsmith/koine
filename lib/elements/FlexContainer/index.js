import styled from 'styled-components';

const FlexContainer = styled.div`
  display: flex;
  flex-direction: column;
`;

export default FlexContainer;
