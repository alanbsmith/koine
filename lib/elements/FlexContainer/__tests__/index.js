import React from 'react';
import FlexContainer from '../index';
import renderer from 'react-test-renderer';


describe('FlexContainer Element', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <FlexContainer />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
