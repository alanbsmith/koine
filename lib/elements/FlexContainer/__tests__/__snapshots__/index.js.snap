// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexContainer Element renders correctly 1`] = `
<div
  className="sc-bdVaJa bPGLpq"
/>
`;
